database/gormdb.go

```diff
@@ -20,7 +19,7 @@ var (
 	ErrEmptyGroup = errors.New("cannot create group without users")
 	// ErrDuplicateGroup is returned when trying to create a group with the same
 	// name as a previously registered group.
-	ErrDuplicateGroup = status.Error(codes.InvalidArgument, "group with this name already registered")
+	ErrDuplicateGroup = connect.NewError(connect.CodeInvalidArgument, errors.New("group with this name already registered"))
```

Comment:
Seems like we are checking for duplicate groups in both the database package and the web package. See `web/groups.go`:
```go
func (s *QuickFeedService) checkGroupName(courseID uint64, groupName string) error {
```
I don't think this error would ever happen, given the other check. I suggest that we just make it a normal error, since it would not be pushed out to the frontend anyway.